examples/visual-tests/amp-story/amp-story-shopping-rtl.html

```diff
@@ -0,0 +1,207 @@
+<!doctype html>
```

Comment:
It is necessary - rtl is only checked on the top level tags - so we had to create another file to test it properly rather than modifying it to use trl on the subtrees. (we are assuming that user will not require a 'mixed' case , where one page could be RTL and another is LTR)